content_scripts/VirtualCamera.js

```diff
@@ -131,15 +139,40 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
                 }
             }.bind(this));
 
+            // when in normal mode, right click to add a green focus cube to the scene
+            const setFocusTargetCube = (event) => {
+                if (this.isFlying) return;
```

Comment:
Idea for a follow-up PR: can we add some small visual feedback onto the screen to show whether you're in fly mode or not? Could be some text when you press F that says "Camera Mode: Fly (Use WASD to move, F to cancel)" for a few seconds. Or it could be an icon that stays visible somewhere in the corner of the screen that shows that you're in flight mode.

Reply:
Good idea! Maybe a plane icon in the top left corner?

Reply:
That a plane icon could look good!